scheduleit/views/meetings/show.twig

```diff
@@ -136,36 +138,55 @@
           </button>
         </div>
         <div class="modal-body">
-          Are you sure you want to remove this attendee?
+          <label id="delete-attendee-modal-body"></label>
+             Are you sure you want to remove this attendee?
         </div>
         <div class="modal-footer">
           <button type="button" class="btn btn-secondary" data-dismiss="modal">Cancel</button>
-          <button type="button" class="btn btn-danger">Delete</button>
+          <button type="button" class="btn btn-danger" id="delete-attendee-button">Delete</button> 
         </div>
       </div>
     </div>
   </div>
+ 
 {% else %}
   <p>
     No attendees have signed up yet.
   </p>
 {% endif %}
+
 <br>
 <div id="onidInvite">
+  <p>
+    <h2 class="h3">Invite</h2>
+  </p>
   <form enctype="multipart/form-data" method="post">
```

Comment:
Don't need to specify `enctype="multipart/form-data"` since we don't have a file upload in this form.

Reply:
This form could live in the modal I set up. It would make the layout cleaner.
```
<div class="modal fade" id="invite-attendees-modal" tabindex="-1" role="dialog" aria-labelledby="invite-attendees-modal-label" aria-hidden="true">
  <div class="modal-dialog" role="document">
    <div class="modal-content">
      <div class="modal-header">
        <h5 class="modal-title" id="invite-attendees-modal-label">Invite Attendees</h5>
        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <form method="post">
        <div class="modal-body">
         <p>
            Please enter a space separated list of ONIDs
          </p>
          <p>
            <input class="form-control" type="text" name="attendeeOnid" id="onid" placeholder="onid1 onid2..." value="{{attendeeOnid}}">
          </p>
        </div>
        <div class="modal-footer">
          <input type="hidden" name="link" value="{{site_url}}/meetings/invite?key={{meeting.hash}}">
          <button type="button" class="btn btn-secondary" data-dismiss="modal">Cancel</button>
          <button class="btn btn-success">Send Invitations</button>
        </div>
      </form>
    </div>
  </div>
</div>
```

Reply:
i spent a lot of time trying to get modals to work and couldn't so i used the thing that i could get to work.